src/python/CRABClient/JobType/UserTarball.py

```diff
@@ -90,8 +90,20 @@ def addFiles(self, userFiles=None, cfgOutputName=None):
             self.tarfile.add(os.path.join(basedir, BOOTSTRAP_CFGFILE_DUMP), arcname=BOOTSTRAP_CFGFILE_DUMP)
 
         #debug directory
-        configtmp = tempfile.NamedTemporaryFile(delete=True)
-        configtmp.write(str(self.config))
+        #configtmp = tempfile.NamedTemporaryFile(delete=True)
+        #configtmp.write(str(self.config))
+        #configtmp.flush()
+        #psetfilename = getattr(self.config.JobType, 'psetName', None)
+        #if not psetfilename == None:
+        #    self.tarfile.add(psetfilename,'/debug/originalPSet.py')
+        #else:
+        #    self.logger.debug('Failed to add pset to tarball')
+        #self.tarfile.add(configtmp.name, '/debug/crabConfig.py')
+        #configtmp.close()
```

Comment:
Actually you commented on outdated code, I force-pushed the cleaned up version of this file after the PR was made. Should probably start mentioning that in the comments.